fingerprint2.js

```diff
@@ -56,7 +60,7 @@
           if (typeof that.options.preprocessor === 'function') {
             componentValue = that.options.preprocessor(pair.key, componentValue)
           }
-          this.data.push({key: pair.key, value: componentValue})
+          keys.data.push({key: pair.key, value: componentValue})
```

Comment:
Ouch, this must have been a merge error. Thanks for fixing this

Reply:
Closure-compiler complained loudly about that one.